### DISCUSSION TITLE
@types/node@16.9.6 build error

### DISCUSSION THREAD

**Discussion by lva98** [DefinitelyTyped/DefinitelyTyped#56043] [Issues with a @types package]
@types/node@16.9.6 causes error when build a project with "yarn tsc --project tsconfig.json"

```
node_modules/@types/node/buffer.d.ts:97:14 - error TS2661: Cannot export 'Buffer'. Only local declarations can be exported from a module.

97     export { Buffer };
                ~~~~~~
Found 1 error.
```

Tsc version is `v4.2.4` and ts-node `v.9.1.1`. Workaround: downgrade @types/node@16.9.6 to @types/node@15.6.0

[a comment by an automated account is not shown]

**Comment by andrevmatos** [comment]
[This workaround](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/25342#issuecomment-865153251) seems to work, but it's not good to skip all libs checks just because `@types/node` has some incompatibility. I got this while upgrading this type definitions in light of recent NodeJS LTS release, but I still didn't manage to find a proper fix.
